rasa/shared/core/events.py

```diff
@@ -246,6 +246,21 @@ def do_events_begin_with_session_start(events: List["Event"]) -> bool:
     )
 
 
+def reduce_event(event: Dict[Text, Any]) -> Dict[Text, Any]:
+    """Reduce event details to the minimum necessary to be structlogged.
+
+    Args:
+        event: The event to be reduced.
+
+    Returns:
+        A reduced copy of the event.
+    """
+    reduced_event = copy.deepcopy(event)
+    if "parse_data" in reduced_event:
+        del reduced_event["parse_data"]
```

Comment:
Was there any particular reason to just delete `parse_data` and not summarise it? (maybe just the first 1-3 results)